lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -49,9 +49,12 @@ def tracer(self) -> opentelemetry.trace.Tracer:
 
 If you are upgrading from `charm_tracing` v0, you need to do the following steps (assuming you already
 have the newest version of the library in your charm):
-1) If you have a dependency on `opentelemetry-exporter-otlp-proto-grpc` in your project for charm tracing, replace
-it with `opentelemetry-exporter-otlp-proto-http>=1.21.0`.
+1) If you have a dependency on `opentelemetry-exporter-otlp-proto-grpc` in your project for charm tracing, replace with:
+
+`opentelemetry-exporter-otlp-proto-http>=1.21.0`.
+
 2) Update your `tracing_endpoint` property to point at `self.tracing.otlp_http_endpoint()`
```

Comment:
```suggestion
2) Update the charm method referenced to from `@trace`, to return from `TracingRequirer.otlp_http_endpoint()` instead of `grpc_http`. For example:

<insert diff>

```

Reply:
Nice idea, added that :)